pkg/datapath/linux/requirements.go

```diff
@@ -144,7 +145,7 @@ func CheckMinRequirements() {
 				log.WithError(err).Warn(errMsg)
 			}
 		}
-		if err := probeManager.CreateHeadersFile(); err != nil {
+		if err := probes.CreateHeaderFiles(filepath.Join(option.Config.BpfDir, "include/bpf"), probes.ExecuteHeaderProbes()); err != nil {
 			log.WithError(err).Fatal("BPF check: NOT OK.")
```

Comment:
This is a rather strange error string, we should probably make it more specific.